pkg/sql/delegate/show_database_indexes.go

```diff
@@ -16,34 +16,48 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 )
 
+// delegateShowDatabaseIndexes implements SHOW INDEX FROM DATABASE, SHOW INDEXES
+// FROM DATABASE, SHOW KEYS FROM DATABASE which returns all the indexes in the
+// given or current database.
 func (d *delegator) delegateShowDatabaseIndexes(
 	n *tree.ShowDatabaseIndexes,
 ) (tree.Statement, error) {
+	name, err := d.getSpecifiedOrCurrentDatabase(n.Database)
+	if err != nil {
+		return nil, err
+	}
+
 	getAllIndexesQuery := `
 SELECT
 	table_name,
-	index_name,
+	info.index_name,
 	non_unique::BOOL,
 	seq_in_index,
 	column_name,
 	direction,
 	storing::BOOL,
-	implicit::BOOL`
+	implicit::BOOL,
+  idx.is_invisible::BOOL`
```

Comment:
Consider `idx.is_invisible::BOOL AS hidden` instead to keep the column names consistent, not just with the `invisible -> hidden` rename I'm advocating, but also for consistency within this table. The other boolean columns don't have an `is_` prefix here.